node/src/accountant/scanners.rs

```diff
@@ -1839,105 +2291,143 @@ mod tests {
             .pending_payable_dao(pending_payable_dao)
             .build();
         let mut pending_payable_fingerprint = make_pending_payable_fingerprint();
-        pending_payable_fingerprint.rowid_opt = Some(rowid);
+        pending_payable_fingerprint.rowid = rowid;
         pending_payable_fingerprint.hash = hash;
 
-        subject.confirm_transaction(pending_payable_fingerprint, &Logger::new("test"));
+        subject.confirm_transactions(vec![pending_payable_fingerprint], &Logger::new("test"));
+    }
+
+    #[test]
+    fn confirm_transactions_does_nothing_if_none_found_on_the_blockchain() {
+        let mut subject = PendingPayableScannerBuilder::new().build();
+
+        subject.confirm_transactions(vec![], &Logger::new("test"))
+
+        //mocked payable DAO didn't panic which means we skipped the actual process
     }
 
     #[test]
-    fn confirm_transaction_works() {
+    fn confirm_transactions_works() {
         init_test_logging();
-        let test_name = "confirm_transaction_works";
         let transaction_confirmed_params_arc = Arc::new(Mutex::new(vec![]));
         let delete_pending_payable_fingerprint_params_arc = Arc::new(Mutex::new(vec![]));
```

Comment:
Conventionally, this would be called `delete_fingerprints_params_arc`, or possibly `pending_payable_dao_delete_fingerprints_params_arc` if two different mocks each had a `delete_fingerprints` method.